CMakeLists.txt

```
@@ -14,6 +14,10 @@ set(ALUMINUM_VERSION_MINOR 5)
 set(ALUMINUM_VERSION_PATCH 0)
 set(ALUMINUM_VERSION "${ALUMINUM_VERSION_MAJOR}.${ALUMINUM_VERSION_MINOR}.${ALUMINUM_VERSION_PATCH}")
 
+if (NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE Release)
```

I recommend setting the cache variable instead. 
```suggestion
  set(CMAKE_BUILD_TYPE Release CACHE STRING "Configuration type" FORCE)
```